packages/cloudflare-workers-bindings-extension/src/add-binding.ts

```diff
@@ -0,0 +1,329 @@
+import {
+	Disposable,
+	env,
+	ExtensionContext,
+	QuickInput,
+	QuickInputButton,
+	QuickInputButtons,
+	QuickPickItem,
+	Uri,
+	window,
+	workspace,
+} from "vscode";
+import { getConfigUri } from "./bindings";
+import { importWrangler } from "./wrangler";
+
+class BindingType implements QuickPickItem {
+	constructor(
+		public label: string,
+		public description?: string,
+		public detail?: string,
+		public iconPath?: Uri
+	) {}
+}
+
+export async function multiStepInput(context: ExtensionContext) {
```

Comment:
Could you give this a more descriptive name?

Reply:
[cc97bed](https://github.com/cloudflare/workers-sdk/pull/7591/commits/cc97bed3cb7af655d8f05c2e2441b60cda85c1c3)